internal/kafka/internal/routes/route_loader.go

```diff
@@ -248,14 +249,8 @@ func (s *options) buildApiBaseRouter(mainRouter *mux.Router, basePath string) er
 
 	adminKafkaHandler := handlers.NewAdminKafkaHandler(s.Kafka, s.AccountService, s.ProviderConfig, s.ClusterService)
 	adminRouter := apiV1Router.PathPrefix("/admin").Subrouter()
-	rolesMapping := map[string][]string{
-		http.MethodGet:    {auth.KasFleetManagerAdminReadRole, auth.KasFleetManagerAdminWriteRole, auth.KasFleetManagerAdminFullRole},
-		http.MethodPatch:  {auth.KasFleetManagerAdminWriteRole, auth.KasFleetManagerAdminFullRole},
-		http.MethodDelete: {auth.KasFleetManagerAdminFullRole},
-	}
-	adminRouter.Use(auth.NewRequireIssuerMiddleware().RequireIssuer([]string{s.Keycloak.GetConfig().OSDClusterIDPRealm.ValidIssuerURI}, errors.ErrorNotFound))
-	adminRouter.Use(auth.NewRolesAuhzMiddleware().RequireRolesForMethods(rolesMapping, errors.ErrorNotFound))
-	adminRouter.Use(auth.NewAuditLogMiddleware().AuditLog(errors.ErrorNotFound))
```

Comment:
Why was this middleware removed?

Reply:
do you think we should temporarily keep it here?

Reply:
Yes, I think the `AuditLogMiddleware` configuration should be kept there. 
```
	adminRouter.Use(auth.NewAuditLogMiddleware().AuditLog(errors.ErrorNotFound))
```

Reply:
oh. sorry. good catch!